### PR TITLE
[do not merge] Allow separate pulp instances for pulp_push and pulp_sync

### PR DESCRIPTION
**Pull request by twaugh**
Allow different pulp instances for `pulp_push` and `pulp_sync`. New configuration parameters `pulp_sync_secret` and `pulp_sync_registry_name` allow separate configuration for `pulp_sync`
plugin. If not set, it will use the `pulp_secret` and `pulp_registry_name` values.

### CODE REVIEW

File: osbs/build/build_request.py [hunk 1]
```diff
@@ -691,7 +698,8 @@ def render(self, validate=True):
 
                           ('postbuild_plugins',
                            'pulp_sync',
-                           'pulp_secret_path'):
+                           'pulp_sync_secret_path'):
```
Comment (twaugh) [inline osbs/build/build_request.py]: Oh, wait, this is wrong.

**Comment by twaugh** [comment]
To make this easier to review, I'll re-factor the commits.

**Comment by twaugh** [comment]
In fact, I'll put the fixes in a separate pull request, and this pull request here can be for having multiple pulp instances.

**Comment by TomasTomecek** [comment]
looks good to multipulp

**Comment by twaugh** [comment]
I've re-based this, but let's hold off on merging it unless it seems to be needed.

**Comment by jpopelka** [comment]
LGTM

**Comment by twaugh** [comment]
Dropping this as it's not needed.

**Pull request closed without merging**